ietf/static_src/css/utilities.scss

```diff
@@ -10,3 +10,7 @@
 .u-max-text-width {
     max-width: 50rem;
 }
+
+.no-js .no-js-hide {
+    display: none;
+}
```

Comment:
Not really a utility thing, not sure where to put this 🤔

Reply:
Maybe a `no-js.scss` file?

Reply:
It seemed like a utility to me (though I didn't add the `u-` prefix), but whatever 😅 

I have added the no-js file.